data_science/terraform/modules/dlami/variables.tf

```diff
@@ -1,62 +1,33 @@
-variable "name" {
-  description = "Name of the ASG to create"
-}
+variable "name" {}
+variable "instance_type" {}
 
-variable "subnet_list" {
-  type = "list"
+variable "hashed_password" {
+  default = "sha1:5310f21e370d:a4d66e725c179218638c21c03d83933aa066db2d"
```

Comment:
Not sure, bate are these potentially creds in VCS? Maybe instead use TF's data kms_secret?